### ISSUE TITLE
Repo says msrv is 1.41.1 but CI tests 1.56

### ISSUE THREAD

**Issue by stevenroose** [BlockstreamResearch/rust-secp256k1-zkp#83]
I noticed that @sanket1729 's musig branch isn't building in 1.41.1, I didn't try master, but since it's a secp-sys issue, I would expect master to also not build.

Did the msrv bump to 1.56? Then the readme is outdated.

**Comment by apoelstra** [comment]
Yep, oops. We bumped the MSRV in #75 but forgot to update the README.
